.github/workflows/workflow.yml

```
@@ -388,3 +388,30 @@ jobs:
       - run: |
           nix develop .#ox-minimal -c ocaml --version
           nix develop .#ox-minimal -c make test-ox
+
+  utop-dev-tool-test:
+    name: Test that the utop dev tool can be built and run
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v5
+      - uses: nixbuild/nix-quick-install-action@v34
+        with:
+          nix_conf: ${{ env.EXTRA_NIX_CONFIG }}
+      - uses: nix-community/cache-nix-action@v6
+        with:
+          primary-key: |
+            nix-${{ runner.os }}-${{ github.job }}-${{ hashFiles('**/*.nix', '**/flake.lock') }}
+          restore-prefixes-first-match: nix-${{ runner.os }}-${{ github.job }}-
+          gc-max-store-size-linux: 2G
+      - name: Set up a project and build it with alice in a nix shell
```

This name doesn't seem accurate to the step, also at least this part is wrong :)

```suggestion
      - name: Set up a project and build it with dune in a nix shell
```